tasks/models.py

```diff
@@ -1,3 +1,27 @@
 from django.db import models
-
+from org.models import Org
+from users.models import User
 # Create your models here.
+class Task(models.Model):
+    org = models.ForeignKey(Org, on_delete=models.CASCADE)
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    social_media_platform = models.CharField(max_length=30)
+    description = models.CharField(max_length=300)
+    share_link = models.CharField( max_length=30)
+    share_text = models.CharField(max_length=65536)
+    share_img = models.ImageField( upload_to='uploads/tasks/share_img', null=False)
```

Comment:
```suggestion
    share_link = models.CharField( max_length=30, null=True)
    share_text = models.CharField(max_length=65536, null=True)
    share_img = models.ImageField( upload_to='uploads/tasks/share_img', null=True)
```

Reply:
the null =True is necessary because different social media has different requirements. 
Ex: whatsapp needs img and text while linkedin needs only link.

Reply:
Oh, okay sir! . Actually I only thought for facebook and instagram, now I get your point!, Thanks!